client/layout/masterbar/logged-in.jsx

```diff
@@ -127,4 +131,4 @@ const MasterbarLoggedIn = React.createClass( {
 } );
 
 // TODO: make this pure when sites can be retrieved from the Redux state
-export default connect( null, { setNextLayoutFocus }, null, { pure: false } )( MasterbarLoggedIn );
+export default connect( null, { setNextLayoutFocus }, null, { pure: false } )( decorate( MasterbarLoggedIn, 'MasterbarLoggedIn' ) );
```

Comment:
We were going to make it optional because depending on where you wrap the component the display name could change (localize changes it, for instance).